lms/djangoapps/instructor_task/tasks.py

```diff
@@ -173,7 +173,7 @@ def get_student_responses(entry_id, xmodule_instance_args):
 @task(base=BaseInstructorTask, routing_key=settings.ORA2_RESPONSES_DOWNLOAD_ROUTING_KEY)
 def get_ora2_responses(entry_id, xmodule_instance_args):
     """
-    Generate a CSV of ora2 responses and push it to S3.
+    Generate a CSV of ora2 responses with anonymous id,  and push it to S3.
```

Comment:
Now that the method is parameterized to handle both anonymous id reports and reports with emails attached, is this comment still true?  I might be missing something, but I think both types of reports hit this method.